auth/gcloud/aio/auth/session.py

```diff
@@ -335,7 +339,17 @@ async def get(
             timeout: float = 10,
             params: Optional[Mapping[str, Union[int, str]]] = None,
             stream: bool = False,
+            auto_decompress: bool = True,
         ) -> Response:
+            if not auto_decompress and not stream:
+                warnings.warn(
+                    'the requests library always decompresses responses when '
+                    'outside of streaming mode; when audo_decompress is '
```

Comment:
s/audo_decompress/auto_decompress/

Reply:
Also can you elaborate on why we want a warning here and not raising an error? Perhaps to maintain backwards compatibility?